__tests__/todo.spec.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from '@testing-library/react-native';
+import { build, fake } from '@jackfranklin/test-data-bot';
+import { firebase as mockFirebase } from '../src/firebase/config';
+
+import Todo from '../src/screens/todo';
+
+// mockFirebase.firestore().useEmulator('localhost', 8080);
+
+const userDataBuilder = build('User Data', {
+  fields: {
+    uid: fake((f) => f.internet.password()),
+    fullName: fake((f) => f.name.findName()),
+    email: fake((f) => f.internet.email()),
+  },
+});
+const user = userDataBuilder();
+
+const dataBuilder = build('Data', {
+  fields: {
+    word: fake((f) => f.lorem.word()),
+  },
+});
+const data = dataBuilder();
+
+const addItem = (targetInput, itemText) => {
+  fireEvent.changeText(targetInput, itemText);
+  fireEvent(targetInput, 'submitEditing');
+};
+
+afterEach(() => cleanup());
```

Comment:
Are you sure this doesn't run automatically? I think on jest it is not needed.

Reply:
Check docs for TL and try it myself, looks like you are right it does run automatically